### DISCUSSION TITLE
Uploaded css stylesheet missing from GitHub page

### DISCUSSION THREAD

**Discussion by Hannah-Ogunyinka** [community/community#23788] [New to GitHub]
<p>Hi all! Really struggling to work out why my uploaded css stylesheet is not being included into my GitHub page.<br>
I’ve double checked the details in link and there doesn’t seem to be a problem on that side.  Any suggestions on what I could try would be gratefully welcomed.</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/capturelifejournal/mock-profile-page" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/55428584/181079341-71ec48ed-5f35-4ba7-8c4e-3c687dd9c38f.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/capturelifejournal/mock-profile-page" target="_blank" rel="nofollow noopener">capturelifejournal/mock-profile-page</a></h3>

<p>Contribute to capturelifejournal/mock-profile-page development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by Gisgar3** [comment]
<p>Hey, welcome to the <strong>GitHub Support Community</strong>! I was able to navigate to your <code>mock-profile-page</code> and it seems that the CSS stylesheet (<code>styles.css</code>) is working properly. It’s possible that <strong>GitHub Pages</strong> didn’t immediately update when you pushed the source to your repository.</p>
<p>Hope this helps! If you have any further issues, don’t hesitate to reply!</p>

**Comment by Hannah-Ogunyinka** [comment, marked as answer]
<p>Oh wow! it is actually working now!!!<br>
It wasn’t all of yesterday.  Today I only edited the link to “CSS/styles.css” rather than “css/styles.css”.  While the chrome developer browser was forgiving of the inconsistency, Github Pages however, is not.  Good to know for future uploads.</p>
